tests/autopilot_pool_expand/aut_pool_expand_test.go

```diff
@@ -0,0 +1,215 @@
+package tests
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	torpedo_api "github.com/portworx/torpedo/drivers/api"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	. "github.com/portworx/torpedo/tests"
+)
+
+var (
+	testNameSuite            = "AutPoolResize"
+	timeout                  = 30 * time.Minute
+	scaleTimeout             = 2 * time.Hour
+	workloadTimeout          = 5 * time.Hour
+	retryInterval            = 30 * time.Second
+	unscheduledResizeTimeout = 10 * time.Minute
+)
+
+var ruleResizeBy50IfPoolUsageMoreThan50AddDisk = torpedo_api.AutopilotRule{
+	Name: "storage-pool-add-disk",
+	ConditionExpressions: []torpedo_api.AutopilotRuleCondition{
+		{
+			Key:      torpedo_api.PxPoolAvailableCapacityMetric,
+			Operator: torpedo_api.LabelSelectorOpLt,
+			Values:   []string{"80"},
```

Comment:
the value `80` here does not match with rule name `ruleResizeBy50IfPoolUsageMoreThan50AddDisk`